kotlinx-coroutines-core/common/test/flow/operators/DropWhileBusyTest.kt

```diff
@@ -0,0 +1,90 @@
+package kotlinx.coroutines.flow.operators
+
+import kotlinx.coroutines.TestBase
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.*
+import kotlinx.coroutines.withVirtualTime
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class DropWhileBusyTest : TestBase() {
+
+    @Test
+    fun `elements are dropped during slow consumption`() = withVirtualTime {
```

Comment:
We don't use back-quotes test names in this project. Please use `testXxx` naming convention in all the tests here. Also, the tests here would be easier to read and review (less scrolling to do) without all the empty lines inside the corresponding test functions.